modules/xfem/test/tests/solid_mechanics_basic/tests

```diff
@@ -51,6 +51,7 @@
     map = false
     # XFEM requires --enable-unique-ids in libmesh
     unique_id = true
+    allow_warnings = true
```

Comment:
Why do we need to allow warnings for this test?

Reply:
Actually, it looks like there are a bunch of tests with allow_warnings=true. Is this something universal for usage of interaction integrals with solid_mechanics?

Reply:
The warnings are because disp_x, disp_y and disp_z are deprecated. Do you want me to change the code such disp_x etc are deprecated only when used in tensor mechanics?

Reply:
Oh, I see -- can't you just change the DomainIntegral block input to use 'displacements=' for these input files?

Reply:
You probably just need to move 'displacements=' from the Mesh block to GlobalParams, and delete 'disp_x', etc from DomainIntegral.